packages/amplify-category-api/provider-utils/awscloudformation/sync-conflict-handler-assets/sync-conflict-handler-index.js.ejs

```diff
@@ -0,0 +1,35 @@
+exports.handler = async (event, context, callback) => {
```

Comment:
This code is generic and only applies to a default schema. After the walkthrough customers can end up with a non-working code, because they've no ```Post``` type or other things. I think we should add a comment that it is sample code and they must edit it to fit their own schema, at a later time we could look into if we can generate this from their schema.

Also I think that the walkthrough should write a message/warning to the console to notify the customers to make sure they take a look at this file.